docs/ml-features.md

```diff
@@ -1284,6 +1284,61 @@ for more details on the API.
 
 </div>
 
+
+## Imputer
+
+Imputation transformer for completing missing values in the dataset, either using the mean or the 
```

Comment:
Maybe something like "The `Imputer` transformer completes missing values in ..."